<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Datasets</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableWidget_paths">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>250</height>
              </size>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>Path</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Enum</string>
              </property>
             </column>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QToolButton" name="toolButton_add_data">
             <property name="text">
              <string>Add image data</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_struct" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_struct">
             <property name="title">
              <string>Structure Composite</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="1">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>translate y</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>rotate</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QDial" name="dial_rotate">
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>70</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>5000</number>
                  </property>
                  <property name="pageStep">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>2500</number>
                  </property>
                  <property name="wrapping">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDial" name="dial_translatey">
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>70</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>5000</number>
                  </property>
                  <property name="pageStep">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>2500</number>
                  </property>
                  <property name="wrapping">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QDial" name="dial_translatex">
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>70</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>5000</number>
                  </property>
                  <property name="pageStep">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>2500</number>
                  </property>
                  <property name="wrapping">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>translate x</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Mode:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_manual">
                  <property name="text">
                   <string>manual</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_drawnfit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>draw and fit</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QWidget" name="widget_draw_and_fit" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QToolButton" name="toolButton_struct_add">
                   <property name="text">
                    <string>Add</string>
                   </property>
                   <property name="popupMode">
                    <enum>QToolButton::InstantPopup</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButton_3">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButton_4">
                   <property name="text">
                    <string>Abort</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget_structures">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>90</number>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Label</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Mean</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Current layer</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_channel"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="SimpleImageView" name="imageViewROI"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_export">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Export structure data to .tsv</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Visualize" name="vis" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Visualize</class>
   <extends>QWidget</extends>
   <header>impose.gui.visualize</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleImageView</class>
   <extends>QGraphicsView</extends>
   <header>impose.gui.widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_manual</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_draw_and_fit</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_drawnfit</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_draw_and_fit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
